pkg/lib/config/saml.go

```diff
@@ -76,12 +104,15 @@ var _ = Schema.Add("SAMLNameIDFormat", `
 }
 `)
 
-type SAMLNameIDFormat string
-
-const (
-	SAMLNameIDFormatUnspecified  SAMLNameIDFormat = "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"
-	SAMLNameIDFormatEmailAddress SAMLNameIDFormat = "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress"
-)
+var _ = Schema.Add("SAMLBinding", `
```

Comment:
This is unused. I guess you want to use it, right?

Reply:
Oh it is a mistake